package/boot/opensbi/Makefile

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2022 OpenWrt.org
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=opensbi
+PKG_RELEASE:=1.2
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/riscv/opensbi
+PKG_SOURCE_DATE:=2022-12-24
+PKG_SOURCE_VERSION:=6b5188ca14e59ce7bf71afe4e7d3d557c3d31bf8
```

Comment:
There is also a release:
https://github.com/riscv-software-src/opensbi/releases/download/v1.2/opensbi-1.2-rv-bin.tar.xz

When you use the commit it is hard to know that this is actually version 1.2.

This should download the tar.xz file:
````
PKG_NAME:=opensbi
PKG_RELEASE:=1.2
PKG_RELEASE:=1

PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-rv-bin.tar.xz
PKG_SOURCE_URL:=https://github.com/riscv-software-src/opensbi/releases/download/v$(PKG_VERSION)
PKG_HASH:=TODO
````